cpp/modules/luma.gl/core/src/model.cpp

```diff
@@ -50,6 +69,28 @@ Model::Model(std::shared_ptr<wgpu::Device> device, const Model::Options& options
   this->pipeline = std::make_unique<wgpu::RenderPipeline>(std::move(pipeline));
 }
 
-void Model::draw() {
-  // TODO(ilija@unfolded.ai): Implement
+void Model::setAttributes(const std::shared_ptr<WebGPUTable>& table) { this->_attributes = table; }
+
+void Model::draw(wgpu::RenderPassEncoder pass) {
+  if (!this->_attributes) {
+    return;
+  }
+
+  std::vector<BindingInitializationHelper> bindings;
```

Comment:
Does this have to be done every draw? Can we save this as part of the model and update in setAttributes?